### PR TITLE
added tags and adjusted resource limits

### PR DESCRIPTION
**Pull request by amlanc1**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Ran tests locally

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: openshift/api-deploy.yml [hunk 1]
```diff
@@ -64,11 +64,11 @@ objects:
           name: ${NAME}-${COMPONENT}${SUFFIX}-${API_VERSION}
           resources:
             limits:
-              cpu: 80m
-              memory: 96Mi
+              cpu: 400m
```
Comment (shaunmillargov) [inline openshift/api-deploy.yml:67]: How have you come up with these limits Amlan? Are they based on the existing API (Reviews) and how much it requires?

**Review by shaunmillargov — COMMENTED** [review]
I'm curious how you have come up with these new limits Amlan?

File: openshift/api-deploy.yml (hunk 1 above)
Comment (amlanc1) [reply openshift/api-deploy.yml:67]: These limits are based on some existing APIs from Digital forms project...I added some more on top of the existing numbers...

**Review by shaunmillargov — APPROVED** [review]
Thank you

**Pull request merged**